scss/_base.scss

```diff
@@ -137,7 +137,7 @@ $move-map-in-xs: true !default;
                 }
             }
 
-            & > a {
+            & > a,span {
```

Comment:
This works, I'll try the other syntax, sorry - I'm a bit of a SASS newb. I did notice that it compiles to more than what is necessary.